libraries/aws_iam_root_user.rb

```diff
@@ -0,0 +1,28 @@
+# author: Miles Tjandrawidjaja
+class AwsIamRootUser < Inspec.resource(1)
+  name 'aws_iam_root_user'
+  desc 'Verifies settings for AWS root account'
+  example "
+    describe aws_iam_root_user do
+      its('access_key_count') { should eq 0 }
+    end
+  "
+
+  def initialize(conn = AWSConnection.new)
+    @client = conn.iam_client
+  end
+
+  def access_key_count
+    summary_account['AccountAccessKeysPresent']
+  end
+
+  def to_s
```

Comment:
Sure thing it looks like this:
```
  IAM Root
     ✔  User access_key_count should eq 0
```

For whatever reasons only the top two words appear on the first line, then the rest appears on the next. Here is more evidence if you were to just name it `1 2 3`, and  have two tests.
```
  1 2
     ✔  3 access_key_count should eq 0
     ✔  3 access_key_count should eq 0
```

This is something that we can investigate further if it isn't how you pictured the formatting.